var/spack/repos/builtin/packages/legion/package.py

```diff
@@ -476,35 +476,28 @@ def cache_test_sources(self):
         install test subdirectory for use during `spack test run`."""
         self.cache_extra_test_sources([join_path("examples", "local_function_tasks")])
 
-    def run_local_function_tasks_test(self):
-        """Run stand alone test: local_function_tasks"""
+    def test_run_local_function_tasks(self):
+        """Run local_function_tasks"""
```

Comment:
I didn't write the original code, so there's a chance I'm misunderstanding. But I think the reason for the original doc comment is that we're testing building a complete (but small) Legion application against the installed build. Therefore, while it's true that this is the example we're choosing to run, the new doc comment obscures the true purpose of this test.

Something like:

> Build an example (local_functions_test) to ensure that Legion is installed correctly and works for external applications

Reply:
That makes sense.  The docstring needs to be relatively short since it's also intended to be a brief description of the test.  Maybe `build and run external application example`?

You can see an example of how the docstring is incorporated into the output of a test in the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#adding-stand-alone-tests.

Reply:
That would be fine. Thanks.